uni-muenster-termine/uni_muenster_termine/items.py

```diff
@@ -1,40 +1,58 @@
-# -*- coding: utf-8 -*-
-
-# Define here the models for your scraped items
-#
-# See documentation in:
+# Models for your scraped items, see:
 # http://doc.scrapy.org/en/latest/topics/items.html
 
 import scrapy
 from scrapy.item import Item, Field
 
+# thanks to https://stackoverflow.com/a/40581523/274811
 class EventItem(scrapy.Item):
     name = scrapy.Field()
     description = scrapy.Field()
     organizer = scrapy.Field()
     url = scrapy.Field()
-    x_tags = scrapy.Field()
     source = scrapy.Field()
-    x_category = scrapy.Field()
     startDate = scrapy.Field()
     endDate = scrapy.Field()
     location = scrapy.Field()
     geo = scrapy.Field()
 
+    def __init__(self):
```

Comment:
- [ ] Ich würde die Initialisierung konsistent halten. Entweder bei jedem Scrapy-Item überall nur die @yyy- oder x-yyy-Attribute in die __init__-Methode eintragen, oder aber direkt bei jedem Scrapy-Item alle Attribute über die __init__-Methode initialisieren. Bisher ist es mal so, mal so gelöst